test/scrabble_test.rb

```diff
@@ -6,8 +6,14 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
```

Comment:
Recommend making this test pass before you work on anything else. Seems like you were starting to worry about the whole problem without thinking of the smallest piece. Make this small piece work first.